src/main/java/fi/aalto/cs/apluscourses/ui/APlusAuthenticationView.java

```diff
@@ -82,7 +82,7 @@ private void createUIComponents() { //
         "What's my token?",
         AllIcons.Ide.External_link_arrow,
         (first, second) ->
-            BrowserUtil.browse(PluginSettings.A_PLUS_BASE_URL + "/accounts/accounts/"));
+            BrowserUtil.browse(authenticationViewModel.getAuthenticationHtmlUrl()));
```

Comment:
sorry, could you please move "What's my token?" to resource.props?

Reply:
will do :+1: